be1-go/hub/standard_hub/message_handling.go

```diff
@@ -19,36 +19,33 @@ import (
 
 const publishError = "failed to publish: %v"
 
-// handleRootChannelPublishMesssage handles an incominxg publish message on the root channel.
+// handleRootChannelPublishMessage handles an incoming publish message on the root channel.
```

Comment:
Some of the typos were already corrected in #1021, you might want to merge master first.

Reply:
Will do thank you